.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build Java
```

Comment:
how is this related to the PR?

Reply:
with these changes we are moving away from AzCi to github actions

Reply:
just next time - separate PRs